arangod/Transaction/Methods.cpp

```diff
@@ -3432,3 +3440,10 @@ Result Methods::replicateOperations(LogicalCollection const& collection,
 
   return res;
 }
+  
+#ifndef USE_ENTERPRISE
+/*static*/ int Methods::validateSmartJoinAttribute(LogicalCollection const&,
+                                                   arangodb::velocypack::Slice) {
+  return TRI_ERROR_NO_ERROR;
+}
```

Comment:
If we have this `#ifndef`, could we remove the other `#ifdef`s around the calls?